cypress/support/commands.js

```diff
@@ -52,6 +52,20 @@ Cypress.Commands.add('setResolution', size => {
   }
 });
 
+Cypress.Commands.add('verifyHashLink', path => {
```

Comment:
I like that you did you research to the point of learning about custom commands, but they are more ideal when they're used across multiple test files. I'd say you can just put this logic into `hashLink.spec.js`